src/containers/ConfirmationContainer/ConfirmationContainer.vue

```diff
@@ -244,7 +244,7 @@ export default {
           ]);
         })
         .catch(err => {
-          ErrorHandler(err, true);
+          ErrorHandler(err, 3, this);
```

Comment:
i dont think you need `this` try importing `import Toasted from 'vue-toasted';` inside the `ErrorHandler` it should work

Reply:
I'm not sure how that would work since Toasted is imported as a Vue installer. That's actually why I pass `this` back to the function so I can reference the vue instance outside of the component.